tests/python/unittest/test_auto_scheduler_feature.py

```diff
@@ -0,0 +1,149 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Test feature extraction"""
+
+import math
+import tempfile
+
+import tvm
+from tvm import te, auto_scheduler
+
+from test_auto_scheduler_common import matmul_auto_scheduler_test
+
+
+def fequal(a, b):
+    return math.fabs(a - b) < 1e-6
+
+
+def test_cpu_matmul():
```

Comment:
Can you add a description for each of these tests making it clear what they're testing? Given how hard coded the asserts are its difficult to tell what we're checking here.